### PR TITLE
Feat/9/projects

### PR DESCRIPTION
**Pull request by wleklinskimateusz**

### CODE REVIEW

File: app/(default)/(home)/page.tsx [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { Technologies } from "./_components/Technologies";
 import { HeroSection } from "./_components/HeroSection";
```
Comment (irosikoni) [inline app/(default)/(home)/page.tsx:3]: empty line

File: app/(default)/(home)/page.tsx [hunk 2]
```diff
@@ -13,8 +15,17 @@ export default async function Home() {
   return (
     <div className="flex flex-col">
       <HeroSection />
-      <PersonalFeatures />
-      <Technologies />
+      {/* <PersonalFeatures />  */}
```
Comment (irosikoni) [inline app/(default)/(home)/page.tsx:18]: leftover

File: app/(default)/projects/(projects)/_components/Feature.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import { ReactNode } from "react";
+
+export const Feature = ({
+  title,
+  children,
+  actions,
+}: {
+  title: string;
+  children: ReactNode;
+  actions?: { label: string; href: string }[];
```
Comment (irosikoni) [inline app/(default)/projects/(projects)/_components/Feature.tsx:19]: move props up

File: app/(default)/projects/(projects)/tetrisly/page.tsx [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+export default function TetrislyPage() {
+  return (
+    <>
+      <Hero
+        title="Tetrisly Design System"
+        logo="/images/projects/tetrisly.svg"
+        className=" bg-[rgba(104,84,227,0.1)]"
+      />
+
```
Comment (irosikoni) [inline app/(default)/projects/(projects)/tetrisly/page.tsx:24]: empty line

File: app/(default)/projects/(projects)/tetrisly/page.tsx [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+                label: "Storybook",
+                href: navigationConfig.tetrisly.react.storybook,
+              },
+              {
+                label: "NPM Package",
+                href: navigationConfig.tetrisly.react.npm,
+              },
+            ]}
+          />
+
```
Comment (irosikoni) [inline app/(default)/projects/(projects)/tetrisly/page.tsx:151]: empty line

File: app/(default)/projects/(projects)/tetrisly/page.tsx [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+          </Skill>
+          <Skill>
+            Plan correct variants of components with an apropriate{" "}
+            <BoldSkill>TypeScript support</BoldSkill>
+          </Skill>
+        </ul>
+      </section>
+    </>
+  );
+}
```
Comment (irosikoni) [inline app/(default)/projects/(projects)/tetrisly/page.tsx:236]: too big component

File: app/(default)/projects/_components/ProjectCard.tsx [hunk 7]
```diff
@@ -1,55 +1,110 @@
...
+                    })
+                  : "In Progress"}
+              </Badge>
+            </div>
+          </CardHeader>
+          <CardContent className="flex grow flex-col justify-between gap-4">
+            <div className="flex gap-4">
+              <div className="my-auto w-2/3">
+                <div className="text-sm">{children}</div>
+              </div>
```
Comment (irosikoni) [inline app/(default)/projects/_components/ProjectCard.tsx:63]: wierd div nesting, cannot do in one?

File: components/ui/badge.tsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
+import * as React from "react"
+import { cva, type VariantProps } from "class-variance-authority"
+
```
Comment (irosikoni) [inline components/ui/badge.tsx:3]: empty line

Comment (wleklinskimateusz) [reply components/ui/badge.tsx:3]: yup, grouped imports

**Pull request merged**